packages/flutter/test/animation/animation_controller_test.dart

```diff
@@ -929,6 +929,49 @@ void main() {
     tick(const Duration(seconds: 1));
     expect(statuses, <AnimationStatus>[AnimationStatus.reverse]);
   });
+
+  test('AnimateBack can runs successfully with just "reverseDuration" property set', () {
+    final List<AnimationStatus> statuses = <AnimationStatus>[];
+    final AnimationController controller = AnimationController(
+      reverseDuration: const Duration(seconds: 2),
+      vsync: const TestVSync(),
+    )..addStatusListener((AnimationStatus status) {
+      statuses.add(status);
+    });
+
+    controller.animateBack(0.8);
+
+    expect(statuses, <AnimationStatus>[AnimationStatus.reverse]);
+    statuses.clear();
+    tick(const Duration(milliseconds: 0));
+    tick(const Duration(seconds: 2));
+    expect(statuses, <AnimationStatus>[AnimationStatus.dismissed]);
+
+    controller.dispose();
+  });
+
+  test('Forward / AnimateTo will throw error without duration', () {
+    final AnimationController controller = AnimationController(
+      vsync: const TestVSync(),
+    );
+
+    controller.dispose();
+
+    expect(() => controller.forward(), throwsAssertionError);
+    expect(() => controller.animateTo(0.8), throwsAssertionError);
+  });
+
+  test('Reverse / AnimateBack will throw error without duration and reverseDuration', () {
+    final AnimationController controller = AnimationController(
+      vsync: const TestVSync(),
+    );
+
+    controller.dispose();
+
+    expect(() => controller.reverse(), throwsAssertionError);
```

Comment:
can you verify it throws FlutterError and its content?

Reply:
done!